Library/Homebrew/livecheck/livecheck.rb

```diff
@@ -289,7 +307,11 @@ def run_checks(
 
           next version_info if version_info.is_a?(Hash) && version_info[:status] && version_info[:messages]
 
-          next status_hash(formula_or_cask, "error", [no_versions_msg], full_name: use_full_name, verbose: verbose)
+          latest_info = status_hash(formula_or_cask, "error", [no_versions_msg], full_name: use_full_name,
+                                                                                 verbose:   verbose)
+          latest_info[:resources] = resource_version_info if check_for_resources
+
+          next latest_info
```

Comment:
The `meta` key is present even when you don't pass `--verbose`.
```
➜ brew lc -r --json influxdb
[
  {
    "formula": "influxdb",
    "status": "error",
    "messages": [
      "Unable to get versions"
    ],
    "meta": {
      "livecheckable": true
    },
    "resources": [
      {
        "resource": "pkg-config-wrapper",
        "version": {
          "current": "0.2.11",
          "latest": "0.2.12",
          "newer_than_upstream": false,
          "outdated": true
        },
        "meta": {
          "url": {
            "original": "https://github.com/influxdata/pkg-config/archive/refs/tags/v0.2.11.tar.gz",
            "processed": "https://github.com/influxdata/pkg-config.git"
          },
          "livecheckable": false
        }
      },
      {
        "resource": "ui-assets",
        "version": {
          "current": "2.4.0",
          "latest": "2022-09-16",
          "newer_than_upstream": false,
          "outdated": true
        },
        "meta": {
          "url": {
            "original": "https://github.com/influxdata/ui/releases/download/OSS-Master/build.tar.gz",
            "processed": "https://github.com/influxdata/ui.git"
          },
          "livecheckable": false
        }
      }
    ]
  }
]
```

Reply:
Are you sure that you are getting this output ? I ran the same command:

```bash
brew lc -r --json influxdb
```

and got the expected output

```json
[
  {
    "formula": "influxdb",
    "version": {
      "current": "2.4.0",
      "latest": "2.4.0",
      "outdated": false,
      "newer_than_upstream": false
    },
    "resources": [
      {
        "resource": "pkg-config-wrapper",
        "version": {
          "current": "0.2.11",
          "latest": "0.2.12",
          "newer_than_upstream": false,
          "outdated": true
        }
      },
      {
        "resource": "ui-assets",
        "version": {
          "current": "2.4.0",
          "latest": "2022-09-16",
          "newer_than_upstream": false,
          "outdated": true
        }
      }
    ]
  }
]

```

Reply:
> This occurs when you modify the `livecheck` block of the formula to raise an error. I modified `influxdb`'s `livecheck` to give an "Unable to get versions" message, as you can see in the output.

Fixed ✅ Thanks for pointing it out.